code/modules/mob/living/carbon/human/human.dm

```diff
@@ -866,9 +866,13 @@
 			piggyback(target)
 			return
 		//If you dragged them to you and you're aggressively grabbing try to fireman carry them
-		else if(user != target && user.a_intent == INTENT_HELP)
-			fireman_carry(target)
+		else if(user != target)
+			if(user.a_intent != INTENT_HELP)
+			to_chat(user, "<span class='warning'>You need to be in help intent to do that!</span>")
 			return

```

Comment:
The PR was just fine as is before this commit. This will fix nothing and annoy everyone who attempts to strip or fireman carry a human mob with an obnoxious feedback message.